### PR TITLE
[frontend] simple stix object relationship fragment update (#6865)

### LINKED ISSUE #6865: Last created relationship with a relationship displayed as Unknown
- Go to an entity that has a relation with a relationship
- In the entity overview, in the 'last created relationship', the relationship target is displayed as Unknown : 
image
- In the Knowledge tab, nothing is displayed in the 'name' column : 
image

When we click on the relationship line : 
image

### PR DESCRIPTION
**Pull request by ValentinBouzinFiligran**
### Proposed changes

* Simple stix object relationship fragment updated to show from and to value
    * from and to added on fragment

### Related issues

* https://github.com/OpenCTI-Platform/opencti/issues/6865

### Checklist

- [ ] I consider the submitted work as finished
- [ ] I tested the code for its functionality
- [ ] I wrote test cases for the relevant uses case (coverage and e2e)
- [ ] I added/update the relevant documentation (either on github or on notion)
- [ ] Where necessary I refactored code to improve the overall quality

### CODE REVIEW

**Comment by Archidoit** [comment]
you should rather use 'representative' in the query to avoid this long list for each entity type
(see toolBarContainersQuery for instance)

**Comment by ValentinBouzinFiligran** [comment]
From data/Relationships
image

With Linked entities 
image

On relationship view 
image

**Comment by labo-flg** [comment]
Tested OK ✅

File: opencti-platform/opencti-front/src/private/components/common/stix_core_relationships/SimpleStixObjectOrStixRelationshipStixCoreRelationshipLine.jsx [hunk 1]
```diff
@@ -433,6 +436,45 @@ const SimpleStixObjectOrStixRelationshipStixCoreRelationshipLineFragment = creat
             }
             ... on StixCoreRelationship {
               relationship_type
+              from {
+                ... on BasicObject {
+                  id
+                  entity_type
```
Comment (labo-flg) [inline opencti-platform/opencti-front/src/private/components/common/stix_core_relationships/SimpleStixObjectOrStixRelationshipStixCoreRelationshipLine.jsx:415]: entity_type is probably not necessary ?
you are not fetching it in `to  { ... onBasicRelationship` for instance.

**Review by SarahBocognano — APPROVED** [review]

**Pull request merged**